content/pages/setting-up-a-github-pages-site-with-jekyll/troubleshooting-jekyll-build-errors-for-github-pages-sites.md

```diff
@@ -146,38 +146,38 @@ This error means that your repository includes a submodule that doesn't exist or
 
 {% data reusables.pages.remove-submodule %}
 
-If you do want to use a submodule, initialize the submodule. For more information, see [Git Tools - Submodules](https://git-scm.com/book/en/v2/Git-Tools-Submodules) in the _Pro Git_ book.
```

Comment:
@akordowski Similarly to comment above, can we ensure the quotation marks remain absent? 💛